Dockerfile

```diff
@@ -41,14 +41,17 @@ RUN pip install redisvl==0.0.7 --no-deps
 # ensure pyjwt is used, not jwt
 RUN pip uninstall jwt -y
 RUN pip uninstall PyJWT -y
-RUN pip install PyJWT --no-cache-dir
+RUN pip install PyJWT==2.9.0 --no-cache-dir
 
 # Build Admin UI
 RUN chmod +x build_admin_ui.sh && ./build_admin_ui.sh
 
 # Runtime stage
 FROM $LITELLM_RUNTIME_IMAGE as runtime
 
+# Update dependencies and clean up
+RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/* 
```

Comment:
what exactly do this do @Jacobh2 ?

Reply:
This upgrades all the Debian dependencies. For example, this is how at least 2 of the major critical issues will be resolved, once the libs have been dreamed stable and are released.